CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Optimizely Objective-C SDK Changelog
 
+## 3.0.2
+February 27th, 2019
+
+This includes a fix to remove type casting requirement for new attribute types
```

Comment:
I think we should be a little more explicit on this one.  Something like eliminate casting to NSObject in Swift.